tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/FociTests.cs

```diff
@@ -55,14 +55,6 @@ public async Task FociSignInSignOutAsync()
             await FociSignInSignOutAsync(labResponse).ConfigureAwait(false);          
         }
 
-        [TestMethod]
-        public async Task FociSignInSignOutMsaUserAsync()
```

Comment:
This is a pretty important test to have deleted. ...

Reply:
yeah, was trying to get some tests to pass, as all the ones w/device code flow are timing out and just slowing down this process.